tools/GeneratePythonLibrary.sh

```diff
@@ -199,6 +198,7 @@ function cleanup {
     kaggle/*.py-e \
     kaggle/api/*.py-e \
     kaggle/*.py.bak
+#    requirements.txt \
```

Comment:
Do we not need requirements anymore?

Reply:
This was actually removing the file. I want to keep it. No need for the comment, though.